common/src/main/java/dev/ftb/mods/ftbchunks/data/FTBChunksTeamData.java

```diff
@@ -56,6 +56,7 @@ public class FTBChunksTeamData {
 	public static final PrivacyProperty NONLIVING_ENTITY_ATTACK_MODE = new PrivacyProperty(new ResourceLocation(FTBChunks.MOD_ID, "nonliving_entity_attack_mode"), PrivacyMode.ALLIES);
 	public static final BooleanProperty ALLOW_EXPLOSIONS = new BooleanProperty(new ResourceLocation(FTBChunks.MOD_ID, "allow_explosions"), false);
 	public static final BooleanProperty ALLOW_MOB_GRIEFING = new BooleanProperty(new ResourceLocation(FTBChunks.MOD_ID, "allow_mob_griefing"), false);
+	public static final BooleanProperty OWNER_UNCLAIM = new BooleanProperty(new ResourceLocation(FTBChunks.MOD_ID, "owner_unclaim"), true);
```

Comment:
Would prefer the default to be true here, so as not to surprise people with a change in behaviour.

Reply:
👍